### DISCUSSION TITLE
Historie und Datenanalyse

### DISCUSSION THREAD

**Discussion by Bootino** [hoylabs/OpenDTU-OnBattery#1054] [Q&A]
Hallo Community,

ich bin auf der Suche nach einer Lösung, um die Daten aus OpenDTUonBattery für die letzten Tage/Monate/Jahre zu sehen? Im besten Fall neben dem Gesamtertrag auch pro WR. 
Warum gibt es allgemein keine Historie? 

2. Frage: Gibt es eine Option zu schauen wie viele Minuten am Tag insgesamt oder unterbrechungsfrei mindestens 1400W/6A anlagen? Zum PV Überschussladen für Elektroautos.

**Comment by spcqike** [comment]
Die meisten Benutzer werden sowas in vorhandene Smart Home Systeme integrieren. Daher, und auch weil es natürlich viele datenpunkte erzeugt die irgendwo gespeichert werden müssen, und der ESP nur wenige MB speichern kann, ist es nicht in der Firmware selbst. 

Ich verwende bspw Node red, influxDB und grafana um mir meine Daten aufzubereiten und anzuzeigen. 

Alternativen wie Home Assistant, FHEM, ioBroker etc sind natürlich auch bei vielen Usern im Einsatz. 

Wenn du das ganze als Überschussladung für E-Autos nehmen willst, kannst du dir auch EVCC angucken. das sollte das auch Smart regeln können. 

Warum ausgerechnet 6A? Regeln die ladeziegel nicht smart mit zwischen 1-10A?

**Reply by Bootino** [reply]
Danke, dann schau ich mir mal Node red, influxDB und grafana an. Gibt es bestimmt gute App in HomeAssistant, dann kann ich dort schauen?

6A-16A bei den meisten Wallboxen, egal ob einphasig oder 3-phasig. 6A ist der Mindeststandard zum Ladestart, laut ISO....
Einphasig kann glaube ich im Tesla eingestellt werden, aber die VW MEB Modelle können alle nur 6-16A.
